common/lifestyle_perks/00_martial_1_strategy_tree_perks.txt

```diff
@@ -44,8 +44,18 @@ parthian_tactics_perk = {
 	character_modifier = {
```

Comment:
Mentions an elephant.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105867667-c8d2d100-600e-11eb-8031-ca4a9d7b0bdc.png)

</details>